tests/concepts/test_craft_tf.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
```

Comment:
Idem, I think it is really light to test only the shape for such a powerful tool. It seems really disproportionate and lack controls on the behavior of the different sub functions

Reply:
Agreed, I added a new `test_classifier` to handle that.